### PR TITLE
Add missing changes from PR #8685 to RB

### PR DESCRIPTION
**Pull request by laemtl**
Add missing changes from PR #8685 to RB (New EEG/MRI Config parent)

### CODE REVIEW

File: raisinbread/RB_files/RB_ConfigSettings.sql [hunk 1]
```diff
@@ -112,12 +113,14 @@ INSERT INTO `ConfigSettings` (`ID`, `Name`, `Description`, `Visible`, `AllowMult
...
 INSERT INTO `ConfigSettings` (`ID`, `Name`, `Description`, `Visible`, `AllowMultiple`, `DataType`, `Parent`, `Label`, `OrderNumber`) VALUES (124,'AWS_S3_Default_Bucket','Default bucket for LORIS to use for accessing files in S3.',1,0,'text',121,'AWS S3 Default Bucket',3);
-INSERT INTO `ConfigSettings` (`ID`, `Name`, `Description`, `Visible`, `AllowMultiple`, `DataType`, `Parent`, `Label`, `OrderNumber`) VALUES (125,'useEEGBrowserVisualizationComponents','Whether to enable the visualization components on the EEG Browser module',1,0,'boolean',39,'Enable the EEG Browser components',4);
+INSERT INTO `ConfigSettings` (`ID`, `Name`, `Description`, `Visible`, `AllowMultiple`, `DataType`, `Parent`, `Label`, `OrderNumber`) VALUES (125,'useEEGBrowserVisualizationComponents','Whether to enable the visualization components on the EEG Browser module',1,0,'boolean',69,'Enable the EEG Browser components',4);
```
Comment (CamilleBeau) [inline raisinbread/RB_files/RB_ConfigSettings.sql:116]: I can't see this change made on the original PR, but I may be missing it?

Comment (laemtl) [reply raisinbread/RB_files/RB_ConfigSettings.sql]: Updated, TY!

File: raisinbread/RB_files/RB_ConfigSettings.sql [hunk 2]
```diff
@@ -112,12 +113,15 @@ INSERT INTO `ConfigSettings` (`ID`, `Name`, `Description`, `Visible`, `AllowMult
...
 INSERT INTO `ConfigSettings` (`ID`, `Name`, `Description`, `Visible`, `AllowMultiple`, `DataType`, `Parent`, `Label`, `OrderNumber`) VALUES (124,'AWS_S3_Default_Bucket','Default bucket for LORIS to use for accessing files in S3.',1,0,'text',121,'AWS S3 Default Bucket',3);
-INSERT INTO `ConfigSettings` (`ID`, `Name`, `Description`, `Visible`, `AllowMultiple`, `DataType`, `Parent`, `Label`, `OrderNumber`) VALUES (125,'useEEGBrowserVisualizationComponents','Whether to enable the visualization components on the EEG Browser module',1,0,'boolean',39,'Enable the EEG Browser components',4);
+INSERT INTO `ConfigSettings` (`ID`, `Name`, `Description`, `Visible`, `AllowMultiple`, `DataType`, `Parent`, `Label`, `OrderNumber`) VALUES 
+(125,'useEEGBrowserVisualizationComponents','Whether to enable the visualization components on the EEG Browser module',1,0,'boolean',39,'Enable the EEG Browser components',4);
```
Comment (CamilleBeau) [inline raisinbread/RB_files/RB_ConfigSettings.sql:117]: I think this should still be on one row

**Review by CamilleBeau — APPROVED** [review]
Looks good and manual test went well! Just a small aesthetic comment

**Comment by CamilleBeau** [comment]
@driusan ready for final review / merge

**Comment by driusan** [comment]
@laemtl please include the issue that it resolves in the PR description in the future.

**Pull request merged**